package.json

```
@@ -6,17 +6,17 @@
   "scripts": {
     "ipr-check": "node scripts/check-form tc39/ecma262 \"$TRAVIS_COMMIT\"",
     "prebuild-spec": "git remote rm travis-origin 2>/dev/null ; git remote add travis-origin \"git@github.com:${TRAVIS_REPO_SLUG:-tc39/ecma262}.git\" && git fetch --quiet travis-origin",
-    "build-spec": "YEAR=2021 && git checkout --quiet \"es${YEAR}\" && mkdir -p \"out/${YEAR}\" && cp -R img \"out/${YEAR}\" && ecmarkup --verbose spec.html \"out/${YEAR}/index.html\" --css \"out/${YEAR}/ecmarkup.css\" --js \"out/${YEAR}/ecmarkup.js\" && git checkout --quiet travis-origin/test-travis",
+    "build-spec": "YEAR=2021 && git checkout --quiet \"es${YEAR}\" && mkdir -p \"out/${YEAR}\" && cpr img \"out/${YEAR}/img\" && ecmarkup --verbose spec.html \"out/${YEAR}/index.html\" --css \"out/${YEAR}/ecmarkup.css\" --js \"out/${YEAR}/ecmarkup.js\" && git checkout --quiet travis-origin/test-travis",
     "postbuild-spec": "git remote rm travis-origin",
     "build-head": "npm run build-only -- --lint-spec --strict",
-    "prebuild-only": "npm run clean && mkdir out && cp -R img out",
+    "prebuild-only": "npm run clean && mkdir out && cpr img out/img/",
```

```suggestion
    "prebuild-only": "npm run clean && mkdir out && node -pe \"fs.copyFileSync('img', 'out/img/')\"",
```
maybe this doesn't need a dependency at all?

---

Unfortunately, `fs.copyFile` cannot copy directories so a one-liner is impossible here. I also tried `fs.symlinkSync` but it requires admin privileges on Windows (while hard links and junction points do not).

In addition I would like to note that I intentionally left `build-spec` unported. I see no sence to introduce another package (`mkdirp` for `mkdir -p`) for a script used by CI and useless for a repo user.

---

I found that `fs.symlinkSync` totally ignores its `type='junction'` argument on non-Windows systems so it can be used without checks and restrictions. 

Thanks to it, this PR finally got rid of third party dependencies.

---

we very much want to copy here, and not make a symlink, so i don't think this is viable